src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx

```diff
@@ -604,7 +551,7 @@ const AddReportDrawerContent: React.FC = () => {
                 alt="status value"
               />
             </div>
-            <StatusValueSelect />
+            <AddDescription />
```

Comment:
Follow the naming rules of previous components. e.g. StatusValueSelect
So you might use `StatusDescriptionAdd`? Or maybe you can come up with a better name.

Reply:
I changed the name of the component to StatusDescriptionAdd.